package.json

```diff
@@ -5,17 +5,22 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon index.js -e ejs"
   },
   "author": "Rocktim Saikia",
   "license": "MIT",
   "dependencies": {
     "axios": "^0.19.0",
     "ejs": "^2.6.1",
-    "express": "^4.16.4"
+    "express": "^4.16.4",
+    "webpack": "^4.41.0"
```

Comment:
@PranjalAgni can you remove the webpack dependencies from the  as we don't need it anymore !

Reply:
Yup :)